src/test/java/hw2/FirstAndFinalCommonSteps.java

```diff
@@ -6,7 +6,7 @@
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 
-public class firstAndFinalCommonSteps {
+public class FirstAndFinalCommonSteps {
```

Comment:
The pull request should not contain any other changes except changes according to the HW3